app/views/templates/_form.html.erb

```diff
@@ -52,6 +52,7 @@
                 title: @template.name,
                 showImport: !@template.locked,
                 showPreview: true,
+                showHostSelector: false,
```

Comment:
Maybe it's not specific to a preview, so we can call it a `host_based?` :thinking:

Reply:
@ezr-ondrej thanks!
The default for showHostSelector is true, so I don't think we should have another function for it, it seems to me not necessary and more complicated.
But I don't mind changing it if you can explain why it's better :) thanks!

Reply:
because right now, no template has the host selector in the preview, but we need report templates not to have it, but others (provisioning etc..) to have the selector visible.